src/tamil/AllTamAbs.gf

```diff
@@ -0,0 +1,9 @@
+--# -path=.:../abstract:../common:prelude
+
+abstract AllTamAbs =
+  Lang,
+  Extend
+  ** {
+    fun sing_V2: V2;
+  } ;
+}
```

Comment:
Remove this last brace, it causes syntax error when trying to compile. (Weird that it is there, I imagine this is just copied from AllMayAbs.) As of now, this file is not really used for anything, but once the grammar develops more, you can put language-specific stuff here for e.g. testing purposes.